benchmark/crypto/aes-gcm-throughput.js

```diff
@@ -8,10 +8,10 @@ var bench = common.createBenchmark(main, {
 });
 
 function main(conf) {
-  var message = (new Buffer(conf.len)).fill('b');
+  var message = (Buffer.allocUnsafe(conf.len)).fill('b');
```

Comment:
Maybe remove the extra parenthesis as well, since it's no longer a `new` call...